doc/content/gateways/gateway-claiming/make-gateways-claimable.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Make a Gateway Claimable"
+description: ""
+weight: 1
+distributions: ["Enterprise", "Cloud"]
+--- 
+
+This guide explains the process of making a gateway claimable. When a gateway is claimable, someone else can claim it securely.
+
+<!--more-->
+
+## Prerequisites
+
+1. An gateway in The Things Stack Cloud. [See instructions]({{< ref "/gateways/adding-gateways" >}})
```

Comment:
```suggestion
1. A gateway registered in The Things Stack Cloud. [See instructions for registering a gateway]({{< ref "/gateways/adding-gateways" >}})
```

Reply:
Why The Things Stack Cloud when it is available for The Things Stack Enterprise too?